vm-main/system/micrortps_agent.service

```diff
@@ -6,7 +6,7 @@ After=network.target
 Type=simple
 User=sad
 Group=sad
-ExecStart=/bin/sh -c ". /opt/ros/galactic/setup_fog.sh; micrortps_agent -b 1000000 -t UDP -r 2020 -s 2019 -n $DRONE_DEVICE_ID -a 127.0.0.1"
+ExecStart=/bin/sh -c ". /opt/ros/galactic/setup_fog.sh; micrortps_agent -b 1000000 -t UDP -r 2020 -s 2019 -n $DRONE_DEVICE_ID"
```

Comment:
Why it has been removed the whitelist IP for local network?

Reply:
It is ok to leave as it is in the PR.

Reply:
Its for the vm-main, not used anymore. In the previous versions, it was not added.